client/src/components/forms/CommentEditor.vue

```diff
@@ -349,6 +353,17 @@ const styleCriteria = ref(
   )
 )
 
+const focusEditor = ref(null)
+
+function focusOnEditor() {
+  console.log(focusEditor.value)
+}
+
+// onMounted(() => {
+//   focusEditor.value.focus()
+// })
+
+
```

Comment:
Since the added code in this file is nonfunctional, I recommend just removing it all. 
```suggestion
```